docs/backup-and-restore.md

```diff
@@ -0,0 +1,73 @@
+Ozone includes backup and restore out of the box. The backup and restore is powered by the powerful and popular [Restic](https://restic.readthedocs.io/en/stable/)  project. Meaning all the backup [backends](https://restic.readthedocs.io/en/stable/030_preparing_a_new_repo.html) supported by Restic can be supported by Ozone.
+
+We wrap Restic in a convience project to allow it to seamlessly intergrate with Ozone and if you are curious about the details you can take a look at the wrapper [project](https://github.com/mekomsolutions/restic-compose-backup)
+
+The backup and restore process is broken into two different services;
+
+- Backup Service (Enabled by default)
+- The restore service, which can be run on time to restore from  a backup generated by the backup service
+
+###  Backup and Restore when using the Ozone helper scripts
+These instructions assume you are already in the Ozone Scripts directory --> see the start Documentation
+#### Backup to Local Path
+```
+export BACKUP_PATH=/tmp/backup
+```
+`./start.sh`
+#### Backup to S3
+To setup the S3 bucket and user following the instructions in the Restic [docs](https://restic.readthedocs.io/en/latest/080_examples.html)
+Export the required ENVs
+```
+export AWS_ACCESS_KEY_ID='secret' 
+export AWS_SECRET_ACCESS_KEY='secret' 
+export RESTIC_REPOSITORY='s3:s3.amazonaws.com/ozone-dev-backup'
+export AWS_DEFAULT_REGION='eu-west-1'
+```
+Run the start script
+`./start.sh`
+
+### Restoring
+#### Restoring from Local path
+Export the required ENVs
+```
+export BACKUP_PATH=/tmp/backup
+export COMPOSE_PROFILES='openmrs-restore,odoo-restore'
+export RESTORE="true"
+```
+Run the start script
+`./start.sh`
+#### Restoring from S3
+Export the required ENVs
+```
+export AWS_ACCESS_KEY_ID='secret' 
+export AWS_SECRET_ACCESS_KEY='secret' 
+export RESTIC_REPOSITORY='s3:s3.amazonaws.com/ozone-dev-backup'
+export AWS_DEFAULT_REGION='eu-west-1'
+export COMPOSE_PROFILES='openmrs-restore,odoo-restore'
+export RESTORE="true"
+```
+Run the start script
+`./start.sh`
+
+
+
+
+[Restic](https://restic.readthedocs.io/en/stable/) provides a rich set of features and a backup storage backends, but Ozone supports only a small subset of the features and local path and S3 for the backup storage backends.
```

Comment:
How come we don't just support all the storage backends that Restic supports?

Reply:
The all require different configurations so we will have quite a bit ENV vars added to Ozone unless we just don't include them and just document them for implimenters to add them manually.